### PR TITLE
Submit final revision of LCS1 for review

### PR DESCRIPTION
**Pull request by chorman0773**

### CODE REVIEW

File: publications/LCS1.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
...eeks to publish various specifications pertaining to Lightning Creations Projects. The Specifications are published by Lightning Creations and the LCS Review Committee on behalf of the author of that specification. These specifications will be assigned some number, and can be referred to as LCS followed by the number. For example, the 1st LCS can be referred to as LCS 1. Additionally, LCS Drafts may be done freely, and can be referred to as LCSD followed by the name of person who is writing the Draft, followed by the number that person chose to assign to it. This will be reviewed for publication according to a process layed out in a future LCS. This process is herein referred to as the "LCS Review Process", and will be carried out by nominated members of Lightning Creations, herein referred to as the "LCS Review Committee". The publication of an LCS is not a statement of endorsement of that LCS by either Lightning Creations or the LCS Review Committee. Neither Lightning Creations or the LCS Review Committee provides any warranties of any kind about Published LCS Documents, and may not be held liable for any damages caused, directly or indirectly, through the use of an LCS Document.
```
Comment (rdrpenguin04) [inline publications/LCS1.md]: Make up your mind whether you are going to call it "specifications" or "Specifications".

File: publications/LCS1.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+
+## §2 Structure of an LCS Document
+
+An LCS Document or LCS is formatted as a series of sections, where the _n_th section is indicated by a Heading labeled as §_n _followed by the given name of the section. Additionally, these sections may be divided into a series of subsections, subsections may be further subdivided, etc. There is no limit to the depth or number of these subdivisions. A subdivision, m levels deep, is labeled as §_n<sub>1</sub>_._n<sub>2</sub>_.(...)._n<sub>m</sub>_. Subdivisions of a section should be related to the level directly enclosing it. Further, sections of an LCS Document should be related to the Specification defined in it.
```
Comment (rdrpenguin04) [inline publications/LCS1.md]: Severely check your formatting on this. Several parts of it are not rendering properly, especially "_n_th" (etc).
"sections of an LCS Document should be related to the Specification defined in it" - Ambiguous use of "it". What are you referring to?

File: publications/LCS1.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+
+
+## §3 LCS Review Committee
+
+The LCS Review Committee is a voluntary position appointed by Lightning Creations. At the offering of a position on the LCS Review Committee, the member may decline or accept the position.  LCS Drafts may be subject to Independent Review by a member of the LCS Review Committee, possibly advised by one or more other members, or Committee Review by the entire LCS Review Committee. The process for Independent Review or Committee Review is not laid out in this Document, and will be defined by a later LCS.  Prior to publication, a final revision of the LCS Draft, structured as an LCS Document will be subject to Committee Review. Additionally, this publication will be subject to Committee Review a year from its publication date, and updated terms may be published as a new LCS Document which supersedes this one. The new LCS Document will then come into effect over this one. A separate LCS Document will be published will all active members of the LCS Review Committee.
```
Comment (rdrpenguin04) [inline publications/LCS1.md:37]: No closer? Not particularly important, but would be nice.

File: publications/LCS1.tex [hunk 4]
```diff
@@ -0,0 +1,112 @@
+\hypertarget{lcs-1---definition-of-the-lightning-creations-specification-project}{%
```
Comment (rdrpenguin04) [inline publications/LCS1.tex:1]: Assume all comments on the MD also apply here.

**Review by rdrpenguin04 — CHANGES_REQUESTED** [review]

File: publications/LCS1.md (hunk 2 above)
Comment (chorman0773) [reply publications/LCS1.md]: Antecedents are complicated :man_shrugging:

File: publications/LCS1.md (hunk 3 above)
Comment (chorman0773) [reply publications/LCS1.md:37]: I decided against having a closer. I can't think of one that would be appropriate.

**Review by rdrpenguin04 — APPROVED** [review]

**Pull request merged**